tests/test_modeling_tf_common.py

```diff
@@ -476,7 +476,7 @@ def check_pt_tf_models(tf_model, pt_model):
                         "TFFunnelForPreTraining",
                         "TFElectraForPreTraining",
                         "TFXLMWithLMHeadModel",
-                    ]:
+                    ] + ["TFTransfoXLLMHeadModel"]:
```

Comment:
Doesn't it make more sense to add it in the list?

Reply:
Copied it from above, but you're right! Will fix both.

Reply:
Done!

Reply:
I separate it to highlight the issue for `TFTransfoXLLMHeadModel` is somehow different than in other models. (`losses`). But I am OK for this change.